cli/tests/lang-node.bats

```diff
@@ -91,23 +91,37 @@ teardown() {
     *-linux)
       # Ensure we're getting krb5 from the flox package by first checking
       # installation fails
-      run ! "$FLOX_BIN" activate -- bash "$TESTS_DIR/init/node/krb5.sh"
+      # XXX "$TESTS_DIR/init/node/krb5.sh" is not always present so only run
+      #     once we have confirmed that it exists, and then expect it to fail.
+      if [ -f "$TESTS_DIR/init/node/krb5.sh" ]; then
```

Comment:
blocking to followup: that file should always exist. If it doesn't exist, we should figure out why, not just skip the test

Also I'd prefer using `skip` so it's more visible

Reply:
Ah forgot about `skip` ... I like that. Will author a patch.

Reply:
Or ... not.

I tried this:
```
@@ -96,6 +96,8 @@ teardown() {
       if [ -f "$TESTS_DIR/init/node/krb5.sh" ]; then
         run "$FLOX_BIN" activate -- bash "$TESTS_DIR/init/node/krb5.sh"
         assert_failure
+      else
+        skip "not checking for installation failure of $TESTS_DIR/init/node/krb5.sh"
       fi
```

... but that had the effect of skipping the _entire_ test, so the following test (from `$INPUT_DATA`) did not run:
```
lang-node.bats                                                                                                         
 ✓ flox activate works with npm [1578]                                                                                                                                                                                                        
 ✓ flox activate works with yarn [1814]                                                                                                                                                                                                       
 - install krb5 with node (skipped: not checking for installation failure of /tmp/nix-shell.b2VZLB/flox-cli-tests-Bwo3bt/init/node/krb5.sh) [952]                                                                                             
```

So then I tried putting the skip in a subshell:
```
@@ -96,6 +96,8 @@ teardown() {
       if [ -f "$TESTS_DIR/init/node/krb5.sh" ]; then
         run "$FLOX_BIN" activate -- bash "$TESTS_DIR/init/node/krb5.sh"
         assert_failure
+      else
+        ( skip "not checking for installation failure of $TESTS_DIR/init/node/krb5.sh" )
       fi
```

And then the tests ran, but no skip message was displayed:
```
lang-node.bats
 ✓ flox activate works with npm [1620]
 ✓ flox activate works with yarn [1786]
 ✓ install krb5 with node [3457]
```

So ... discussed with Yannik and agreed to create an issue to review this test specifically with a future commit.

Reply:
Logged #2323 to investigate this further.